circleci/github-release

```diff
@@ -19,6 +17,42 @@ if [[ -z $ARTIFACTS_DIR ]]; then echo "Missing arg2 ARTIFACTS_DIR" && exit 1; fi
 # Set automatically by CircleCI
 : ${CIRCLE_PROJECT_REPONAME?"Missing required env var"}
 REPO=$CIRCLE_PROJECT_REPONAME
+: ${CIRCLE_PROJECT_USERNAME?"Missing required env var"}
+USER=$CIRCLE_PROJECT_USERNAME
 
-echo "Error! Not yet implemented"
-return 1
+if [ ! -e VERSION ]; then echo "Missing VERSION file" && exit 1; fi
+
+# Download github-release script
+echo "Downloading github-release tool"
+curl -sSL -o /tmp/github-release.tar.bz2 https://github.com/aktau/github-release/releases/download/v0.5.2/linux-amd64-github-release.tar.bz2
+tar jxf /tmp/github-release.tar.bz2 -C /tmp/ && sudo mv /tmp/bin/linux/amd64/github-release /usr/local/bin/github-release
+
+TAG=$(head -n 1 VERSION)
+DESCRIPTION=$(tail -n +2 VERSION)
+
+result=$(github-release release -u $USER -r $REPO -t $TAG -n "$TAG" -d "$DESCRIPTION" || true)
+if [[ $result == *422* ]]; then
+  echo "Release already exists for this tag.";
+  exit 0
+elif [[ $result == "" ]]; then
+  echo "Release created.";
+else
+  echo "Error creating release: $result"
+  exit 1
+fi
+
+for f in $ARTIFACTS_DIR; do
+    # treat directories and files differently
+    if [ -d $f ]; then
+        for ff in $(ls $f); do
+            echo -e "uploading $ff"
+            github-release upload -u $USER -r $REPO -t $TAG -n $ff -f $f/$ff
```

Comment:
A failure on this step is already pretty loud (as can be seen here: https://circleci.com/gh/Clever/wag-test/44), so I don't think we need specific error handling